StRoot/StTpcHitMaker/StTpcHitMaker.cxx

```diff
@@ -397,7 +399,7 @@ Int_t StTpcHitMaker::InitRun(Int_t runnumber) {
       for(Int_t row=1;row<=St_tpcPadConfigC::instance()->numberOfRows(sector);row++) {
         Int_t numPadsAtRow = St_tpcPadConfigC::instance()->padsPerRow(sector,row);
         totalSecPads += numPadsAtRow;
-        if (StDetectorDbTpcRDOMasks::instance()->isRowOn(sector,row) &&
+        if (StDetectorDbTpcRDOMasks::instance()->isRowOn(sector,row,1) &&
```

Comment:
Hi @fisyak ,

Quick question before I do the approval. Where is this function defined `isOn(int, int, int)`? I see one here:
https://github.com/star-bnl/star-sw/blob/4641c7dfe27bcb88ee0ff673769e55258ead82a6/StRoot/StDetectorDbMaker/St_tpcRDOMasksC.h#L36

but it only has two arguments. What am I missing?

Reply:
Irakli, this is isRowOn

Reply:
Oh, thanks. My bad.

Reply:
Hi Irakli, there are two implementations of isOn with 2 and 3 arguments.